app/views/shf_application_mailer/acknowledge_received.html.haml

```diff
@@ -12,10 +12,4 @@
       = shf_app_to_html(@shf_application)
   %hr
 
-  - if @shf_application.uploaded_files.size == 0
```

Comment:
Need to also change the specification `spec/mailers/shf_application_mailer_spec.rb`, which is the failing RSpec.
In that file, remove these lines:  

`lines 131 - 134`

```
   it 'no files uploaded' do
        expect(email_sent).to have_body_text(I18n.t('mailers.shf_application_mailer.acknowledge_received.review_after_recevied_files'))
      end
```

Reply:
awesome comment. Dunno how I missed it for 20 days :o